### PR TITLE
Simplepie using Composer

### PR DESCRIPTION
**Pull request by fiammybe**
Use the SimplePie library managed via Composer. Updating to a new simplepie library is now as easy as 'composer update'.

### CODE REVIEW

File: htdocs/libraries/icms/core/HTMLFilter.php [hunk 1]
```diff
@@ -50,29 +50,29 @@ static public function filterHTML($html) {
...
+
         $purified = strpos($html, $fcomment);
         if ($purified !== FALSE) {
             $html = str_replace($fcomment, '', $html);
         }
 
         if ($icmsConfigPurifier['enable_purifier'] !== 0) {
-			require_once ICMS_LIBRARIES_PATH . '/htmlpurifier/HTMLPurifier.standalone.php';
-			require_once ICMS_LIBRARIES_PATH . '/htmlpurifier/HTMLPurifier.autoload.php';
+//			require_once ICMS_LIBRARIES_PATH . '/htmlpurifier/HTMLPurifier.standalone.php';
```
Comment (MekDrop) [inline htdocs/libraries/icms/core/HTMLFilter.php]: I think it would be better to remove these lines - not to comment. Because it's not very usefull.

**Comment by skenow** [comment]
Conflicts now exist - need to resolve these, first

**Comment by skenow** [comment]
It looks like this has been merged already - at least the current composer.json has the simplepie library included and HTMLPurifier has the lines commented out instead of removed. Possibly committed through another merge?

**Comment by fiammybe** [comment]
This is a mistake : the feature is about simplepie, and the changed lines affect the htmlpurifier. Closing PR, and starting over.

**Pull request closed without merging**